### ISSUE TITLE
caddy port conflict (+ ipv6 only)

### ISSUE THREAD

**Issue by sakgoyal** [TeamPiped/Piped-Docker#32]
I have a service already running on port 443 and 80 on my host. but I also want to run piped. but, when I use the docker AIO script, caddy has an error because 443 is already in use. 

The other service is nextcloud running as a snap on the local host and is assigned an IPv6 address. 

I have allocated a different set IPv6 address for Piped and I want run piped in docker, but I dont know how to set up caddy to handle this situation. can anyone help me figure this out?

Basically, I have one IPv6 address for Nextcloud on the machine, and a secondary one I have assigned for Piped. how do I configure caddy/piped to use the secondary IP address so they do not conflict. 

Also, I do NOT want to use IPv4. this machine does not have IPv4 access to the internet.

**Comment by FireMasterK** [comment]
You should use the guide at https://docs.piped.video/docs/self-hosting/#docker-compose-nginx-aio-script, it's for situations where you already have a reverse proxy.

**Comment by sakgoyal** [comment]
@FireMasterK would you mind giving me an example on how to do this? I am not familiar with nginx/reverse proxies.

**Comment by FireMasterK** [comment]
The [docs](https://docs.piped.video/docs/self-hosting/) have now been updated to make it more clear, with configurations in nginx, Apache 2, Caddy, and Traefik.
